samples/iolib/src/main/cpp/player/SimpleMultiPlayer.h

```diff
@@ -60,16 +58,16 @@ class SimpleMultiPlayer : public AudioStreamCallback  {
 
 private:
     // Oboe Audio Stream
-    AudioStream *mAudioStream { nullptr };
+    oboe::AudioStream *mAudioStream { nullptr };
 
     // Audio attributs
     int32_t mChannelCount;
     int32_t mSampleRate;
 
     // Sample Data
     int32_t mNumSampleBuffers;
-    SampleBuffer**          mSampleBuffers;
-    OneShotSampleSource**   mSampleSources;
+    std::unique_ptr<SampleBuffer**> mSampleBuffers;
+    std::unique_ptr<OneShotSampleSource**>   mSampleSources;
```

Comment:
Same as above

Reply:
Made this std::vector<OneShotSampleSource*> as per our discussion.